components/AddButton/AddButton.d.ts

```diff
@@ -36,15 +37,35 @@ export interface AddButtonProps extends Omit<ButtonProps, 'color'> {
           "dangerNoBackground" |
           "roseNoBackground"
   /**
-   *  @default "medium"
-   * The size of button. 
-    */
+  *  @default "medium"
+  * The size of button. 
+  */
   size?: "medium" | "small"
   /**
-   *  @default "small"
+  *  @default "small"
   * The fontSize of button's content.
   */
-  fontSize?: "inherit" | "default" | "small" | "medium" | "large"
+  fontSize?: "inherit" | "small" | "medium" | "large"
+  /**
+  * Override or extend the styles applied to the component
+  */
+  customClass?: string
```

Comment:
This should be renamed to **className**, this is the naming convention implied by material-ui library. We should change in IconButton the functionality that uses this variable, to allow receiving **className** instead of **customClass**